pandas/tests/series/test_arithmetic.py

```diff
@@ -349,6 +349,7 @@ def test_add_list_to_masked_array(self, val, dtype):
         result = [1, None, val] + ser
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.filterwarnings("ignore:evaluating in Python space:UserWarning")
     def test_add_list_to_masked_array_boolean(self):
```

Comment:
side note: ive seen these and wondered why the lists aren't getting wrapped before being passed to the EA method

Reply:
I'll add a todo comment here to that effect